assets/app/scripts/directives/logViewer.js

```diff
@@ -71,8 +71,8 @@ angular.module('openshiftConsole')
 
 
 
-
-            var updateScrollLinks = function() {
+            // is just toggling show/hide, nothing else.
+            var toggleScrollLinks = function() {
```

Comment:
I'm OK with that, although "set" to me implies you are passing in the new value. Maybe `updateScrollLinksVisibility`? Wordy, but descriptive.